src/components/DrawerContainer.js

```diff
@@ -56,24 +61,24 @@ class DrawerContainer extends Component {
           }
           <List>
             { user &&
-              <ListItem button component={Link} to={PAGES.account.path}>
+              <ListItem button component={Link} to={PAGES.account.path} onClick={() => this.gaPageView(PAGES.account.path)}>
                 <ListItemIcon><AccountCircleIcon/></ListItemIcon>
                 <ListItemText primary={PAGES.account.label} />
               </ListItem>
             }
 
             { user && user.isModerator &&
-              <ListItem button component={Link} to={PAGES.moderator.path}>
+              <ListItem button component={Link} to={PAGES.moderator.path} onClick={() => this.gaPageView(PAGES.moderator.path)}>
                 <ListItemIcon><CheckCircleIcon /></ListItemIcon>
                 <ListItemText primary={PAGES.moderator.label} />
               </ListItem>
             }
 
-            <ListItem button component={Link} to={PAGES.tutorial.path}>
+            <ListItem button component={Link} to={PAGES.tutorial.path} onClick={() => this.gaPageView(PAGES.tutorial.path)}>
```

Comment:
instead than hard code an event per link, can you instead use some kind of change listener (route, history,)?